api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -197,6 +197,15 @@ message JwtProvider {
   // Specify the clock skew in seconds when verifying JWT time constraint,
   // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
   uint32 clock_skew_seconds = 10;
+
+  // Enable cache.
+  bool enable_token_cache = 11;
```

Comment:
Please describe the cache algorithm
// Cache is a LRU cache, its side is specified by `token_cache_size`
// Only cache good JWT, 
// Cache JWT only removed when it is expired.  Or removed by LRU purge.

Reply:
Done.